modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase

```diff
@@ -828,6 +828,201 @@ definition {
+	@priority = 5
+	test CanUserWithAccountManagerRoleViewAndManageChannelDefaults {
+		property custom.properties = "jsonws.web.service.paths.excludes=";
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And the admin creates two business types of accounts associated with an organization and assigns account 1 to a user without any role") {
+			JSONOrganization.addOrganization(organizationName = "Test Organization");
+
+			for (var index : list "1,2") {
+				JSONAccountEntry.addAccountEntry(
+					accountEntryName = "Commerce Account ${index}",
+					accountEntryType = "Business");
+
+				Account.openAccountsAdmin();
+
+				LexiconEntry.gotoEntry(rowEntry = "Commerce Account ${index}");
+
+				Account.assignOrganizations(
+					accountName = "Commerce Account ${index}",
+					assigneeName = "Test Organization");
+			}
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account 1",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "useramea@liferay.com",
+				userFirstName = "useramfn",
+				userLastName = "useramln",
+				userScreenName = "useramsn",
+				userSiteMembership = "Minium");
+
+			JSONOrganization.assignUserToOrganization(
+				organizationName = "Test Organization",
+				userEmailAddress = "useramea@liferay.com");
+		}
+
+		task ("When the user navigates to the Account Management page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "useramea@liferay.com",
+				userLoginFullName = "useramfn useramln");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/account-management");
+		}
+
+		task ("Then can view only the unlinked account 1 listed") {
+			for (var index : list "1,2") {
+				AssertElementNotPresent(
+					key_entryName = "Commerce Account ${index}",
+					locator1 = "CommerceNavigation#ENTRY_NAME");
+			}
+
+			AssertElementPresent(
+				key_tableEntry = "Commerce Account 1",
+				locator1 = "LexiconTable#TABLE_ENTRY");
+
+			AssertElementNotPresent(
+				key_tableEntry = "Commerce Account 2",
+				locator1 = "LexiconTable#TABLE_ENTRY");
+		}
+
+		task ("And cannot see the account entry's ellipsis operation") {
+			AssertElementNotPresent(locator1 = "Icon#SVG_ELLIPSIS");
+		}
+
+		task ("When the Account Manager role is given to the user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			User.openUsersAdmin();
+
+			User.assignOrgRoleCP(
+				roleTitle = "Account Manager",
+				userScreenName = "useramsn");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "useramea@liferay.com",
+				userLoginFullName = "useramfn useramln");
+		}
+
+		task ("Then can view account 1 listed with a link") {
```

Comment:
After Account Manager role is given, the user can see both accounts and their defaults, please update these last two parts to assert it for both accounts, and use Control Panel to navigate to Accounts Admin.